/*
 * This file was automatically generated by EvoSuite
 * Mon Jan 11 04:23:29 GMT 2021
 */

package com.iluwatar.datamapper;

import org.junit.Test;
import static org.junit.Assert.*;
import com.iluwatar.datamapper.DataMapperException;
import org.evosuite.runtime.EvoRunner;
import org.evosuite.runtime.EvoRunnerParameters;
import org.junit.runner.RunWith;

@RunWith(EvoRunner.class) @EvoRunnerParameters(mockJVMNonDeterminism = true, useVFS = true, useVNET = true, resetStaticState = true, separateClassLoader = true) 
public class DataMapperException_ESTest extends DataMapperException_ESTest_scaffolding {

  @Test(timeout = 4000)
  public void test0()  throws Throwable  {
      DataMapperException dataMapperException0 = new DataMapperException("com.iluwatar.datamapper.DataMapperException");
  }
}
